kafka-rest/src/test/java/io/confluent/kafkarest/KafkaRestConfigTest.java

```diff
@@ -94,14 +94,14 @@ public void getConsumerProperties_propagateMetrics_defaultClusterId() {
   @Test
   public void getConsumerProperties_propagateMetricsProperties() {
     Properties properties = new Properties();
-    properties.put(reporter_config("telemetry.bootstrap.servers"), "broker1:9092");
+    properties.put(reporter_config("bootstrap.servers"), "broker1:9092");
```

Comment:
nit, bootstrap server might be a bit confusing since it's not an actual config, instead maybe use and actual config such as `api.key` or alternatively some placeholder such as `some.config`